config/config.go

```diff
@@ -53,8 +53,11 @@ type Ports struct {
 // Config represents pmm-agent's static configuration.
 //nolint:maligned
 type Config struct {
-	ID      string `yaml:"id"`
-	Address string `yaml:"address"`
+	ID       string `yaml:"id"`
+	Address  string `yaml:"address"`
+	Username string `yaml:"username"`
+	Password string `yaml:"password"`
+	Port     uint16 `yaml:"port"`
```

Comment:
Please rename it to `ListenPort` or something else. We have another port as a part of `Address`.
Maybe rename `Address` to `ServerAddress`.